### ISSUE TITLE
UI delay when installing a package to ASP.NET website

### ISSUE THREAD

**Issue by kartheekp-ms** [NuGet/Home#8736]
Linked to ADO Bug [959481](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/959481)

**Comment by kartheekp-ms** [comment]
Reference to old project system API in client [code](https://github.com/NuGet/NuGet.Client/blob/c1f6915918b82c096bbd666bd9c18528f1f70630/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/WebSiteProjectSystem.cs#L43) seems to causing this UI delay. Working with Web Tools Engineering Team offline to learn if there is any better way of retrieving Assembly References and adding reference from file for a Website project system.

**Comment by kartheekp-ms** [comment]
Blocked by ADO Bug [959481](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/959481)

**Comment by kartheekp-ms** [comment]
Details about this issue are documented in this [comment](https://github.com/aspnet/websdk/issues/1036#issue-512582162)

**Comment by nkolev92** [comment]
The AzDO issue has been won't fixed, closing this per that. 

@kartheekp-ms Let me know if you think I missed something.

**Comment by kartheekp-ms** [comment]
I am okay closing this issue because the AzDO issue was closed as won't fix.
